site/src/pages/components/date-picker/RangeDatePickerExample.demo.tsx

```diff
@@ -0,0 +1,24 @@
+import { RangeDateField } from 'bold-ui'
+import React, { useState } from 'react'
+
+function RangeDatePickerExample() {
+  const [value, setValue] = useState<Date>()
+
+  const handleChange = (selectedDate: Date) => setValue(selectedDate)
+
+  return (
+    <RangeDateField
+      error=''
+      icon='calendarOutline'
+      label='Period'
+      onChange={function noRefCheck() {}}
```

Comment:
You can change this line for `onChange={console.log}` so the event is logged on the developer tools